drivers/modem/hl7800.c

```diff
@@ -4079,7 +4077,6 @@ static int modem_reset_and_configure(void)
 	LOG_ERR("Unable to configure modem");
 	ictx.configured = false;
 	set_network_state(HL7800_UNABLE_TO_CONFIGURE);
-	modem_reset();
```

Comment:
Is this intentional to remove the reset on errors?

Reply:
Yes, we don't want to hold the modem in reset if initialization fails. This will allow an app to still communicate with the modem.